src/utils/fs.ts

```diff
@@ -8,3 +9,15 @@ export const exists = async (path: string): Promise<boolean> => {
         return false;
     }
 };
+
+export const softFileURLToPath = (fileName: string): string => {
+    if (!fileName.startsWith("file://")) {
+        return fileName;
+    }
+
+    try {
+        return fileURLToPath(fileName);
+    } catch (_) {
+        return fileName;
+    }
+};
```

Comment:
Pulled out of "error-snippets" because it is used inside "browser/stacktrace" too